adapters/quick/src/main/scala/caliban/GraphiQLAdapter.scala

```diff
@@ -0,0 +1,87 @@
+package caliban
+
+import zio.http._
+
+object GraphiQLAdapter {
```

Comment:
Maybe weird to call it "adapter" since it's not really what it does with our definition of adapters. Maybe just `GraphiQLHandler`?